recipes/recipe/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "openmolcas" %}
+{% set version = "21.10" %}
+
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/Molcas/OpenMolcas/-/archive/v{{ version }}/OpenMolcas-v{{ version }}.tar.gz 
+  sha256: 46f30c47a9f30e030cf1a4ac8b73964da81657ea9093654da057a21c7fac63f3
+
+build:
+  number: {{ build }}
+  skip: true  # [not linux]
```

Comment:
if you have _any_ guidance from the upstream on supported python versions, probably worth adding to the skip... or just make one up (e.g. `py < 37`), as it might reduce the broken builds

Reply:
[The correct one would be `py >= 27`](https://gitlab.com/Molcas/OpenMolcas/-/wikis/quick%20start) but given that conda-forge is on python 3.6 (or is it 3.7 already?) I'm not sure that's worth encoding here?